examples/fastify/index.ts

```diff
@@ -0,0 +1,34 @@
+import { GraphQLServer } from 'graphql-yoga';
+import fastify from 'fastify';
+import { GraphQLObjectType, GraphQLSchema, GraphQLString } from 'graphql';
+
+const app = fastify({ logger: true });
+
+const graphQLServer = new GraphQLServer({
+    schema: new GraphQLSchema({
+        query: new GraphQLObjectType({
+            name: 'Query',
+            fields: {
+                greetings: {
+                    type: GraphQLString,
+                    resolve: () => 'Hello World!'
+                }
+            }
+        })
+    })
+})
+
+app.route({
+    url: '/graphql',
+    method: ['GET', 'POST', 'OPTIONS'],
+    handler: (req, res) => {
+        return graphQLServer.requestListener(req as any, res.raw);
```

Comment:
@ardatan are you sure it will work with fastify plugins etc? It skips the `res.json()` `res.end()` and other. Let's say Fastify depends on `res.end()` only and does not listen to `res.raw.end()` so potentially the `fastify-prometheus` or `fastify-whatever` won't detect that the response was send.

Reply:
Actually you're right. In helix, that's why we use reply.send instead of raw response. I'll try sth else.